docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml

```diff
@@ -136,39 +136,41 @@ question: |
   Final Review With Cover Page
 subquestion: |
   
-  Below is your ${comma_and_list(al_download_titles)} document with the
-  cover page that we will deliver to ${courts[0]}.
+  Below is your document with the 
+  cover page that will be delivered to the ${courts[0]}.
   
   Click "Back" if you need to make any changes.  
-  Click "Send to court" to deliver it.
+  Click "Send to the ${courts[0]}" to deliver it.
   
   % if users[0].email:
   **By sending this form to the court, you agree that you
   will get notices from the court by email.** If you do not want to receive
   notices by email, click "Back", download your form, and send it to court
-  by mail or by dropping it off.
+  by mail or drop it off.
```

Comment:
I'm not sure it flows here, there are a lot of prepositions? What about:

```
download your form, and mail it to the court or drop it off.
```

Reply:
Changed in new commit.